.rubocop.yml

```diff
@@ -37,6 +37,10 @@ Style/Alias:
 Style/CommandLiteral:
   EnforcedStyle: percent_x
 
+# Checks uses of global variables.
```

Comment:
I would add a TODO comment saying that this is disabled until we agree on it. 

Also, I would remove what the cop does, as I already mentioned in another PR. This is documented in Rubocop, the documentation can change and we do not want to update this when that happens.

Reply:
Then let's agree on it first. We already collect such cops in the [trello card](https://trello.com/c/JAeV3sf1/631-5-p6-fix-rubocop-process).

Reply:
@bgeuken 

> Then let's agree on it first. We already collect such cops in the trello card.

How is this related with what I said? 🤔  I just want that you add a comment with the reason why this is disabled. Just write that you disabled because we don't know if we want this or not. If afterwards we agree on disabling it, we just update the comment. In the meanwhile (and we don't know how long the meanwhile will take), if someone looks at this file he will know why this is disabled. 😉

Reply:
@Ana06, this cop is not disabled, we are only allowing some variables. Like I said in the PR #4256 I prefer to do it in a different PR. This comment was added only because the others cops have the same kind of comment. But when we agree about the affected cops I will add the correct comment in the correspondent PR.

Reply:
The comment was for all the cops affected in this PR (I could have repeated the same comment, but I thought it was clear). I would add a comment explaining why this need to be in the conf file. If it is disabled, why or a TODO comment. And if it is changing the default config, you can still write why it is needed.

If something is wrong, I would prefer that you do not add more code with the same problem. But if you want to do it in another PR I am ok with it. But please add why these cops are in this file, because it may be not that obvious. :smile: